web/src/components/localization-select/localization-select-complex.tsx

```diff
@@ -82,9 +91,11 @@ const LocalizationSelectComplex = ({ locale, onLocaleChange }: Props) => {
                   key={item}
                   className={classNames('list-item-wrapper', {
                     highlighted: index === highlightedIndex,
-                  })}>
+                  })}
+                  style={{ borderBottomColor: userLanguages.length > 0 && index === userLanguages.length-1 ? "#333" : "#f3f2f0" }}
```

Comment:
Could you use the `classNames` library to add a class that does? Also use the colors we have defined in `vars.css` in the css for this component

Reply:
I nearly always used CSS libraries. `classNames` is a simple and powerful one, especially conditionals!
Done